lib/helium/client/device_configurations.rb

```diff
@@ -0,0 +1,64 @@
+module Helium
+  class Client
+    module DeviceConfigurations
+
+      def device_configurations
+        DeviceConfiguration.all(client: self)
+      end
+
+      def device_configuration(id)
+        DeviceConfiguration.find(id, client: self)
+      end
+
+      def device_configuration_configuration(device_config)
+        path = "/device-configuration/#{device_config.id}/configuration"
+        response = get(path)
+        configj = JSON.parse(response.body)["data"]
+        config = Configuration.new(client: self, params: configj)
+        return config
+      end
+
+      def device_configuration_device(device_config)
+        path = "/device-configuration/#{device_config.id}/device"
+        response = get(path)
+        configj = JSON.parse(response.body)["data"]
+
+        if configj.dig("type") == "sensor"
+          device = Sensor.new(client: self, params: configj)
+        elsif configj.dig("type") == "element"
+          device = Element.new(client: self, params: configj)
+        end
+
+        return device
+      end
+
+      def create_device_configuration(device, configuration)
+        path = "/device-configuration"
+
+        body = {
+          data: {
```

Comment:
I was on the fence about how to deal with seemingly one-off things like this.  Probably worth a refactor once we add in `metadata` support.